DeskReserve_Tests/DeskReserve_Tests.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
```

Comment:
There is something going on here with these multiple .sln files. We should have only one